extendedmodlog/eventmixin.py

```diff
@@ -1174,17 +1172,16 @@ async def on_message_edit(self, before: discord.Message, after: discord.Message)
         fmt = "%H:%M:%S"
         if embed_links:
             embed = discord.Embed(
-                description=before.content,
+                description=f"**Message edited in {before.channel.mention}** [Jump to message]({after.jump_url})",
```

Comment:
This line won't be affected by people trying to use translation features. You mus encapsulate the english text in _() outside of the f-strings here.